.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,58 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Is the .idea directory needed in the repository? Are these files not automatically generated?

Reply:
not need. i will delete it

Reply:
yes, this files auto created